### PR TITLE
Fix cloud-init local configuration

### PR DESCRIPTION
**Pull request by valentindavid**

### CODE REVIEW

File: lib/build-image.sh [hunk 1]
```diff
@@ -84,7 +84,7 @@ EOF
         shadow_unit="$(base64 -w0 etc-shadow.mount)"
         cat <<EOF >"${TMPDIR}/pc/cloud.conf"
 #cloud-config
-datasource_list: [NoCloud]
+datasource_list: [NoCloud,None]
```
Comment (alfonsosanchezbeato) [inline lib/build-image.sh]: Should be enough with `[None]`, at least that worked in my last tests and we have already changed snapd tests to this.

Comment (valentindavid) [reply lib/build-image.sh]: OK, I will test it.

Comment (valentindavid) [reply lib/build-image.sh]: That worked. Thank you.

**Review by alfonsosanchezbeato — APPROVED** [review]
LGTM, thanks!

**Pull request merged**